### PR TITLE
Grunt connect

### PR DESCRIPTION
**Pull request by ollwenjones**
If this repo uses grunt for so much, why not also provide contributors with an easy dev server?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .baseDir.*
 .idea
+node_modules/
```
Comment (ollwenjones) [inline .gitignore]: Also wasn't sure why `node_modules` wasn't gitignored ?

**Comment by ollwenjones** [comment]
These tests are failing because the browsers aren't starting...

**Comment by justindujardin** [comment]
I didn't include it because I normally just launch from WebStorm directly.  I suppose this makes sense.

Please rebase on the latest master to fix the travis build issue for PRs.

**Comment by ollwenjones** [comment]
@justindujardin Thank you.

I 'merged' the latest master in instead of rebasing, since I was taught to "never rebase published commits." If you really want the rebase I can do that though, let me know. I think it will show up as a single merge commit in your history either way though.

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "grunt-bump": "0.7.0",
     "grunt-cli": "0.1.13",
     "grunt-contrib-clean": "0.7.0",
+    "grunt-contrib-connect": "^0.11.2",
```
Comment (justindujardin) [inline package.json]: please lock this version by removing the `^`

**Comment by justindujardin** [comment]
Looks good to me, just lock the version and we can merge it. :+1:

**Comment by ollwenjones** [comment]
Version locked. Thanks.

**Comment by justindujardin** [comment]
Thanks!

**Pull request merged**